keycloak/keycloak-admin-ui
# Add filter capability to keys section in realm settings

## Motivation

Towards #639

## Verification Steps


1. Go to Realm Settings -> Keys
2. Click on the "keys list" tab.
3. Click on the select menu above the keys list and choose a new filter type.
4. Verify the listed providers are listed correctly based on the filter type. 

## Checklist:


## Additional Notes
